src/__tests__/fixture.ts

```diff
@@ -0,0 +1,43 @@
+export const executionFromOusideTypedData = (to: string) => ({
```

Comment:
$nitpic

Technically this is the typed data for the OutsideExecution struct so `outsideExecutionTypedData` would be more accurate.

https://github.com/starknet-io/SNIPs/blob/main/SNIPS/snip-9.md#1-build-the-outsideexecution-struct

Reply:
ah yep, will refactor